p2pserver/net/netserver/netserver.go

```diff
@@ -64,6 +64,7 @@ type NetServer struct {
 	ConnectingNodes
 	PeerAddrMap
 	Np *peer.NbrPeers
+	sync.RWMutex
```

Comment:
document which field this mutex protect ? only find it is used in Connect function

Reply:
I think this block:

	connCount := this.GetOutConnectingListLen()
	if connCount > config.DefConfig.P2PNode.MaxConnOutBound {
		log.Warnf("Connect: out connections(%d) reach the max limit(%d)", connCount,
			config.DefConfig.P2PNode.MaxConnOutBound)
		return errors.New("connect: out connections reach the max limit")
	}

in Connect function needs to be protected, and I'v changed sync.RWMutex to sync.Mutex

Reply:
If so, I highly suggest you naming it `connectLock` instead of embedding struct field.

Reply:
ok~